angular/core/components/notifications/notifications.haml

```diff
@@ -9,6 +9,6 @@
       %ul.selector-list
         %li.selector-list-header{translate: "notifications.header", role: "heading"}
         %li.selector-list-item.media{ng-class: "{'lmo-active': !notification.viewed}", ng-repeat: "notification in notifications() | orderBy: '-createdAt' track by notification.id"}
-          %a.selector-list-item-link{class: "navbar-notifications__{{notification.kind()}}", lmo-href-for: "notification.relevantRecord()", lmo-href-action: "{{notification.actionPath()}}", ng-click: "broadcastThreadEvent(notification)"}
+          %a.selector-list-item-link{class: "navbar-notifications__{{notification.kind()}}", lmo-href-for: "notification.actionModel()", lmo-href-action: "{{notification.actionPath()}}", ng-click: "broadcastThreadEvent(notification)"}
```

Comment:
We're in the somewhat odd position of determining the href of the link from outside the item's controller, so the notification needs to know about where it's linking to in `actionModel()`

This could be improved, probably.